inc/projecttask.class.php

```diff
@@ -614,7 +614,10 @@ function projecttasktemplate_update(value) {
                $("#textfield_name'.$rand_name.'").val(data.name);
 
                // set textarea description
-               $("#description'.$rand_description.'").val(data.description);
+               $("#description'.$rand_description.'").html(data.description);
+               if (tasktinymce = tinymce.get("description'.$rand_description.'")) {
+                  tasktinymce.setContent(data.description.replace(/\r?\n/g, "<br />"));
+               }
```

Comment:
1. Unless I miss something, you set textarea contents by redefining html content of textarea and by the setContent method of tinyMce. Only one should be used.

2. As task template is defined from a rich text, replacement of \n to <br> seems useless.

Reply:
it's copy pasta from another parts of the code. Thanks for raising this btw.
I'll try to fix both parts in the morning

Reply:
for point 1, i think it was because before, we had options to set glpi in simple or rich text.
Not the case anymore. So we can drop some code